insights/parsers/pci_rport_target_disk_paths.py

```diff
@@ -152,10 +152,15 @@ def parse_content(self, content):
         self.__host_channel_id_lun_attributes = []
 
         for line in content:
+            if (('host' not in line) or ('pci' not in line) or
+                    ('target' not in line) or ('block' not in line)):
+                raise ParseException(BADWD.format(line))
+
             line_sp = list(filter(None, line.strip().split('/')))
             temp_pci = {}
+            rport = None
```

Comment:
This `rport` should not be initialized out of the following loop, which means the value of it in the former line could be inherited by the following lines in which the `"rport-"` is not found.